README.md

```diff
@@ -107,31 +98,28 @@ export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
 export AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION
 ```
 
-4. Run `terraform init`
+3. Run `terraform init`
 
-5. `terraform plan` and `terraform apply` will now work as expected, as will
+4. `terraform plan` and `terraform apply` will now work as expected, as will
    `terraform destroy`.
 
-6. Test cases also run as part of the pipeline.
+5. Test cases also run as part of the pipeline.
    See [test-terraform-module.yml](https://github.com/honeycombio/terraform-aws-integrations/blob/main/.github/workflows/test-terraform-module.yml)
 
 ### Docs
 
-Docs are autogenerated by running `make generate-docs`, and put
-in [USAGE.md](https://github.com/honeycombio/terraform-aws-integrations/blob/main/USAGE.md).
+Docs located on the Terraform website are autogenerated by running `make generate-docs`, and put in [USAGE.md](https://github.com/honeycombio/terraform-aws-integrations/blob/main/USAGE.md).
```

Comment:
Which docs are these? Let's include a link to where the Terraform-hosted docs are. (I assume Terraform ingests this USAGE.md file somehow and places it somewhere in their docs.)

Reply:
https://registry.terraform.io/modules/honeycombio/integrations/aws/latest

Reply:
the registry does not use usage.md though - we have kept this block the same in all our modules FYI

Reply:
I did see that the Terraform link displays our current main repo README (nifty!)

Reply:
Amended phrasing